app/lib/lnd/walletUnlocker.js

```diff
@@ -115,6 +87,39 @@ class WalletUnlocker {
   // Helpers
   // ------------------------------------
 
+  /**
+   * Establish a connection to the Lightning interface.
+   */
+  async establishConnection() {
+    const { host, cert } = this.lndConfig
```

Comment:
This looks like a complicated method. What happens if we get an exception somewhere there?  Is it handled somewhere so the user is aware that something went wrong?

Reply:
This class is fairly low level, and the caller of this method is expected to handle any errors from it.

We do handle these already and ensure that we pass on appropriate errors to the client where possible. For example, see `onBeforeStartRemoteLnd` where we parse some of the known errors from failed connection attempts.